torch/testing/_internal/distributed/rpc/dist_autograd_test.py

```diff
@@ -2051,3 +2054,45 @@ def backward(ctx, grad):
             # static_grad_values_ref are holding onto views and don't bump the
             # refcount.
             self.assertTrue(p_g == p_a)
+
+@unittest.skipIf(
+    not torch._six.PY3,
+    "Pytorch distributed autograd package " "does not support python2",
+)
+class FaultyAgentDistAutogradTest(FaultyRpcAgentTestFixture):
```

Comment:
Let's add a test to verify that the fixture has set the backend and options properly.